packages/odyssey-react/src/components/Field/types.ts

```diff
@@ -0,0 +1,46 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import type { ReactNode } from "react";
+
+export interface SharedFieldTypes {
```

Comment:
moving shared types into their own module (rather than being exported from the barrel file for `Field`) continues to allow us to share types, however without exporting them as part of our top level public API